src/test/integration/test_integration.py

```diff
@@ -147,10 +145,128 @@ def test_search_species(self):
         result = body['result'][0]
         ranks = set(r['rank'] for r in result['results'])
         self.assertEqual(len(result['results']), 10)
-        self.assertEqual(ranks, {'species'})
+        self.assertEqual(ranks, {'species', 'no rank'})
```

Comment:
Because some docs have `{'rank': 'no rank', 'strain': True}`
Not sure why this wasn't being caught before ...

Reply:
Oh I think I know why, because sorting has been implemented now